sdk/compute/Azure.ResourceManager.Compute/src/Customize/RestOperations/SshPublicKeysRestOperations.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager.Compute.Models;
+
+namespace Azure.ResourceManager.Compute
+{
+    internal partial class SshPublicKeysRestOperations
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public async Task<Response<SshPublicKeyGenerateKeyPairResult>> GenerateKeyPairAsync(string subscriptionId, string resourceGroupName, string sshPublicKeyName, CancellationToken cancellationToken = default)
```

Comment:
If I understand this correctly, the case here is that:
In previous swagger, this request does not take a body.
In the new api-version, it could take a body, but the body is optional.
If this is the case, we should not have these customizations.
Please remove this partial class, we do not really need it.
I will leave comments in other classes to follow up because if you just remove this, you will get compilation errors.